src/logging.cpp

```diff
@@ -14,7 +14,7 @@ FILE* Logging::log_stream = nullptr;
 #define print_func(...) fprintf(Logging::log_stream, __VA_ARGS__)
 #endif
 
-Logging::Logging(ELogLevel level, string file, int line, string function_name)
+Logging::Logging(ELogLevel level, string file, int /*line*/, string function_name)
```

Comment:
no references? And maybe these strings can be const char* so they don't create strings.

Reply:
yeah it's unused, regardless of configuration.

> maybe these strings can be const char* so they don't create strings.
I'm with you, but the repo owner is pretty adamant about one change / one PR... 😏 
I'll toss in another review for it soon-ish.

Reply:
Annoying owner